tests/load-tests/ci-scripts/utility_scripts/playwright-create-users.py

```diff
@@ -0,0 +1,251 @@
+
+
+def get_verification_link(user_email):
+    """
+    This uses mine specific mail setup and will not probably work for
+    anybody else. If you are in that case sitoation, yo need to click all
+    the verification emails manually or automate it to suit your setup.
+    """
+    attempt = 1
+    attempt_max = 3
+    attempt_sleep = 10
+    while True:
+        if attempt > attempt_max:
+            raise Exception(f"Out of tries to get verification email for {user_email}")
+
+        try:
+            fetchmail = subprocess.run(
+                ["fetchmail"], stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+            )
+            if fetchmail.returncode not in (
+                0,
+                1,
+            ):  # 0 = emails downloaded, 1 = no new emails
+                raise Exception(
+                    f"Running 'fetchmail' failed with {fetchmail.returncode}: {fetchmail.stdout}"
+                )
+
+            notmuch_search = subprocess.run(
+                [
+                    "notmuch",
+                    "search",
+                    "--output=files",
+                    "--format=json",
+                    "--sort=newest-first",
+                    f"to:'{user_email}' AND subject:'Verify email for Red Hat account' AND date:'1hour..now'",
+                ],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
+            if notmuch_search.returncode != 0:
+                raise Exception(
+                    f"Running 'notmuch search ...' failed with {notmuch_search.returncode}: {notmuch_search.stderr}"
+                )
+
+            message_files_json = notmuch_search.stdout
+            message_files = json.loads(message_files_json)
+            if len(message_files) != 1:
+                raise Exception(
+                    f"Command 'notmuch search ...' returned unexpected number of files: {message_files}"
+                )
+
+            if not os.path.isfile(message_files[0]):
+                raise Exception("File {message_files[0]} is missing")
+        except Exception as e:
+            print(f"Getting mail failed: {e}")
+            attempt += 1
+            time.sleep(attempt_sleep)
+        else:
+            break
+
+    with open(message_files[0], "r") as fp:
+        msg = email.parser.Parser(policy=email.policy.default).parse(fp)
+    simplest = msg.get_body(preferencelist=("plain"))
+    for line in simplest.get_content().splitlines():
+        line = line.strip()
+        if line.startswith(
+            "https://sso.redhat.com/auth/realms/redhat-external/login-actions/action-token?key="
+        ):
+            return line
+
+
+def click_and_wait_hard(
+    page, clickable, verifier, timeout_ms_start=1000, timeout_ms_max=30000
+):
+    """
+    Try to click at given thingy (probably link) and wait for another
+    thingy to appear, try it multiple times untill it works
+    """
+    timeout = timeout_ms_start
+    while True:
```

Comment:
shouldn't we consider to use some limitation of retries to avoid 
an infinite loop  ?

Reply:
Right. It is slightly below here:
```
           if timeout >= timeout_ms_max:
                raise  # giving up
            timeout *= 2
```